lib-src/rpc2/rp2gen/crout.c

```diff
@@ -217,20 +219,18 @@ void print_struct_func(RPC2_TYPE *t, FILE *where, FILE *hfile, char *name, WHO w
 			printf("RP2GEN: illegal type for RPC2_Struct field: %s\n", (*v)->name);
 			exit(1);
 		}
-		fputs("    ", where);
-		print_unpack_var("ptr->", *v, where, who);
+		print_unpack_var("ptr->", *v, where);
 	}
     fputs("    return 0;\n", where);
     fputs("}\n", where);
 }
-void print_unpack_var(char* prefix, VAR* var, FILE *where, WHO who) {
+void print_unpack_var(char* prefix, VAR* var, FILE *where) {
     extern char* concat();
     char* name, *select, *suffix;
     MODE mode;
     char _iterate[] = "_iterate";
 
     name = concat(prefix, var->name);
-    select = ((who == RP2_CLIENT && mode != NO_MODE) ? "->" : ".");
```

Comment:
Why was this removed, it looks like it is still used in various placed below. f.i. `case RPC2_COUNTEDBS_TAG:`